examples/gpbp_showcase.ipynb

```diff
@@ -65,10 +56,19 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "adm_area.get_facilities(method=\"osm\", tags={\"building\":\"hospital\"})\n",
+    "adm_area.get_facilities(method=\"osm\", tags={\"amenity\":\"hospital\"})\n",
```

Comment:
I changed the tag to `amenity` because I noticed there were almost no hospitals being loaded when the tag was of type `building`.